zerver/webhooks/bitbucket3/view.py

```diff
@@ -436,7 +436,7 @@ def api_bitbucket3_webhook(
     if "eventKey" in payload:
         eventkey = payload["eventKey"].tame(check_string)
     else:
-        eventkey = request.META["HTTP_X_EVENT_KEY"]
+        eventkey = request.headers["X-Event-Key"]
```

Comment:
We should make this not 500 if the header isn't present, too -- this is the one spot that does a direct access, so:
```suggestion
        eventkey = validate_extract_webhook_http_header(request, "X-Event-Key", "BitBucket", fatal=True)
```

Reply:
Changed.